CHANGES

```diff
@@ -65,6 +42,30 @@ Version 7.1.0 - In Development
     - Extend all SOP references to support VDB Points.
     - Combine SOP will not error in flatten-all mode if second has no grids.
 
+    Build:
+    - Improved the CMake build for the OpenVDB Houdini library on Windows.
+      [Reported by Ian Woodward]
+    - Remove some logic for compilers older than Visual Studio 2019.
+    - Fixed a bug in the CMake FindIlmBase/OpenEXR modules which could cause
+      compilers on UNIX systems to fail to find stdlib.h if IlmBase/OpenEXR
+      headers were installed to /usr/include.
+    - CMake now checks the minimum supported MSVC compiler for Windows.
+    - Add gitignore to repository root.
```

Comment:
A couple of these items are tenuously "build":

* Add gitignore to repository root
* Remove some logic for compilers older than Visual Studio 2019

I guess I'd keep all CI, version control and deprecation changes in "improvements". That would make the "build" section focus more narrowly on CMake (how and where something builds) as well as fixing any compiler warnings.

Reply:
done